server/controllers/controller.js

```diff
@@ -1,6 +1,41 @@
 const Model = require('../db-models/database');
 const axios = require('axios');
 
+exports.createGallery = async ({ headerImage, mainImages, moreLikeThisImages }) => {
+  let count = await Model.countDocuments({}, function (err, count) {
+    if (err) {
+        console.log(err);
+    } else {
+        return count++;
+    }
+  });
+
+  count++;
+
+  const gallery = {
+    page: count.toString(),
+    headerImage: headerImage,
+    mainImages: JSON.parse(mainImages),
+    moreLikeThisImages: JSON.parse(moreLikeThisImages)
+  }
+  return Model.create(gallery);
+}
+
+exports.updateGallery = (page, { headerImage, mainImages, moreLikeThisImages }) => {
+  console.log(typeof page);
+  console.log(headerImage);
+  const filter = {
+    page: '101'
+  }
+  const update = {
+    page: '101',
+    headerImage: headerImage,
+    mainImages: mainImages,
+    moreLikeThisImages: moreLikeThisImages
+  };
+  return Model.findOneAndUpdate(filter, update);
+}
+
```

Comment:
Are you simply updating page 101 because its out of the original 1-100 scope and is good for testing your CRUD route? Or would it be better to pass in the page number dynamically from Postman?

Reply:
i saw this comment via email notification shown below. not sure why you saw `page: '101'` at the time.
![image](https://user-images.githubusercontent.com/16806021/116934029-3ea39580-ac19-11eb-9907-74730ef7b4af.png)
you can see in the actual PR im using es6 syntax to set page in both filter and update
![image](https://user-images.githubusercontent.com/16806021/116934137-6692f900-ac19-11eb-85ed-e540ab523914.png)